src/explorer-components/PageCard/index.js

```diff
@@ -8,10 +8,9 @@ const Title = props => {
   const { children, className } = props;
   return (
     <Text
-      className={className}
-      className={styles.title}
-      typeScale="h2"
-      color="white"
+      className={className ? className : styles.title}
```

Comment:
@codeluggage this needs both:
className={`${className} {styles.title}`}

Overriding classes don't replace existing classes. They just add to it. The previous version was bad.... thanks for catching this.